src/Context/AxiosCalls/AxiosCallProvider.jsx

```diff
@@ -62,33 +65,24 @@ const AxiosCallProvider = ({ children }) => {
 
     try {
       const res = await axios.post(url, body, headers);
-      console.log(res.data);
-    } catch (error) {
-      setError(error.message);
-      setShowError(true);
-    }
-  };
-
-  // restore note from trash
-  const restoreFromTrashOnServer = async (noteConfig) => {
-    const { url, body, headers, item } = noteConfig;
-
-    try {
-      const res = await axios.post(url, body, headers);
-      console.log(res.data);
+      //update after adding note
+      dispatch({ type: "notesAfterAddingNew", payload: res.data.notes });
+      console.log("notesAfterAddingNew", res.data);
     } catch (error) {
       setError(error.message);
       setShowError(true);
     }
   };
 
   // remove note
-  const addToTrashFromServer = async (noteConfig) => {
-    const { url, headers, item } = noteConfig;
-
+  const addToTrashOnServer = async (noteConfig) => {
+    const { url, headers } = noteConfig;
+    //
     try {
       const res = await axios.delete(url, headers);
-      console.log(res.data);
+      //update after deleting note
+      dispatch({ type: "notesAfterDelete", payload: res.data.notes });
+      console.log("notesAfterDelete", res.data.notes);
```

Comment:
I would suggest giving the type of dispatch a capital Name Convention otherwise it is good.